app/scripts/components/common/chart/utils.ts

```diff
@@ -92,8 +92,11 @@ export function getFData({
   yKey: string;
   dateFormat: string;
 }): { uniqueKeys: any[], fData: FormattedTimeSeriesData[] } {
+  // Check sensitivity value
+  const collator = new Intl.Collator(undefined, {numeric: true, sensitivity: 'base'});
```

Comment:
Anybody can give me insight if it is reasonable to consider accent or other diacritic marks for sorting? (or any other values for sensitivity? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator/Collator. options are `base`, `case`, `variant`) I've been living accentless life so I have no sense.

Reply:
I imagine that for the majority of use cases we don't need to worry about accents. However, I'd say to disregard accents when sorting. Consider `à` equal to `a`